Jenkinsfile

```diff
@@ -46,6 +47,7 @@ pipeline {
         environment name: 'PROJECT', value: 'kinto-settings'
       }
       steps {
+        sh "cp ${KINTO_DOTENV} .env"
```

Comment:
`KINTO_DOTENV` doesn't appear to be defined.

Reply:
Fixed